src/js/entry.js

```diff
@@ -208,3 +209,25 @@ export function rootApp() {
         );
     }
 }
+
+export function noAccess() {
+    const { store } = spinUpStore();
+    const currPath = location.pathname;
+    const app = currPath.split('/').pop();
+    window.insights.chrome.auth.getUser().then(({ entitlements }) => {
+        const apps = { insights: entitlements.insights.is_entitled };
```

Comment:
This works perfectly for insights, what do you think about including other apps as well?
```JS
window.insights.chrome.auth.getUser().then(({ entitlements }) => {
  // rhel has different entitlements key and URL partial
  entitlements.rhel = entitlements.smart_management;
  const grantAccess = Object.entries(entitlements).filter(([ app, { is_entitled } ]) => {
    // check if app key from entitlements is anywhere in URL and if so check if user is entitled for such app
    return location.pathname.split('/').includes(app) && is_entitled;
  });
  if (grantAccess && grantAccess.length > 0) {
    document.getElementById('root').style.display = 'none';
    render(
      <Provider store={ store }>
        <NoAccess />
      </Provider>,
      document.querySelector('#no-access')
    )
  }
})
```
`location.pathname.split('/').includes(app) && is_entitled` has limitations, if there's going to be URL with `insights/rhel` in future this will return true. But we'll need much complicated logic to calculate such thing. But for now I feel like it is sufficient enough.

Reply:
Yes, that makes sense! I will include this in the commit. Thanks!